tests/test.t

```diff
@@ -181,10 +181,13 @@ array:serializeArrayFromDict[`ScalarTest; scalars_expected_dict];
 
 -1 "<--- Pass correct data --->";
 
-repeated_from_scalars:{@[x;where 10h=type each x;enlist]}
+repeated_from_scalars:{@[x;where any each 4 10h=\:/: type each x;enlist]}
 repeated_scalars:repeated_from_scalars scalars_expected;
 repeated_expected:repeated_scalars,'repeated_scalars;
 
+show scalars_expected;
+show repeated_expected;
+
```

Comment:
You should probably remove these shows.  The tests don't display any debugging, only errors.